R/theme-current.R

```diff
@@ -91,9 +91,12 @@ get_theme <- function() {
 theme_get <- get_theme
 
 #' @rdname get_theme
-#' @param new new theme (a list of theme elements)
+#' @param new new theme (a list of theme elements). Sets theme to the default (theme_grey) if `new` isn't supplied.
 #' @export
 set_theme <- function(new) {
+  if (missing(new)) {
```

Comment:
We generally prefer to take `NULL` as default argument, then you can use the `%||%` operator to assign the default:

```r
new <- new %||% ggplot_global$theme_default
```